### PR TITLE
chore: add test cases

### PR DESCRIPTION
**Pull request by mathur07**

### CODE REVIEW

File: tests/api/test_api_v2.py
Comment (rafiu007) [inline tests/api/test_api_v2.py:173]: This should be "ecosystem" .
and For python this should not be the response.
currently only golang is not supported

**Review by rafiu007 — APPROVED** [review]
LGTM. Suggested a minior change.
Wait for review from @yzainee

**Review by yzainee-zz — APPROVED** [review]
LGTM. I would like to see a thing added though. As the fixed_in field is a list, try adding 2-3 values in the graph mock response and see if you get all the 3 in the response you send back from the api.

**Review by rafiu007 — APPROVED** [review]
LGTM

**Pull request merged**
